github/messages_test.go

```diff
@@ -82,6 +83,54 @@ func TestValidatePayload(t *testing.T) {
 	}
 }
 
+func TestValidateFormPayloadGET(t *testing.T) {
+	payload := `{"yo":true}`
+	signature := "sha1=3374ef144403e8035423b23b02e2c9d7a4c50368"
+	secretKey := []byte("0123456789abcdef")
+
+	req, err := http.NewRequest("GET", "http://localhost/event", nil)
+	q := req.URL.Query()
+	q.Add("payload", payload)
+	req.URL.RawQuery = q.Encode()
+	if err != nil {
+		t.Fatalf("NewRequest: %v", err)
+	}
+	req.Header.Add("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
Since you only ever want one value for Content-Type, use `req.Header.Set` rather than `req.Header.Add`. It doesn't make a difference as long as you do it only once, but still, using `Set` is better form and more readable and consistent code.